### PR TITLE
feat(pie-input): DSW-000 add element internals polyfill

### PR DESCRIPTION
**Pull request by jamieomaguire**
## Describe your changes (can list changeset entries if preferable)
---
"@justeattakeaway/pie-input": minor
---

[Added] - Use the element internals polyfill in pie-input component

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] I have reviewed visual test updates properly before approving
- [x] If changes will affect consumers of the package, I have created a changeset entry.
- [x] If a changeset file has been created, I have used the `/snapit` functionality to test my changes in a consuming application

### Reviewer 1 - Kev
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2 Raouf
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

**Review by raoufswe — APPROVED** [review]

File: packages/components/pie-input/src/index.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ import styles from './input.scss?inline';
 import {
     types, statusTypes, InputProps, InputDefaultPropertyValues,
 } from './defs';
+import 'element-internals-polyfill';
```
Comment (kevinrodrigues) [inline packages/components/pie-input/src/index.ts:17]: Could you add a note to the readme similar to the modal one around the inclusion of this polyfill please?

Comment (jamieomaguire) [reply packages/components/pie-input/src/index.ts:17]: Hey Kevin! Happy to, although I cannot see any notes for modal in the readme or index file. Could you please point me to it?

Comment (kevinrodrigues) [reply packages/components/pie-input/src/index.ts:17]: I thought i saw it there. If it's not there then dont worry for now. We can manually look through components later when removing the polyfills.

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**